Dockerfile

```
@@ -263,7 +243,7 @@ ENV IS_DOCKER_8395080871=1
 
 RUN ldconfig
 WORKDIR /tmp
-ENTRYPOINT ["/opt/conda/bin/fprodents"]
+ENTRYPOINT ["/opt/conda/envs/fprodents/"]
```

I think you still want a program as an entrypoint.

```suggestion
ENTRYPOINT ["/opt/conda/envs/fprodents/bin/fprodents"]
```